### ISSUE TITLE
[FEATURE] Auomating moving/renaming/creating/deleting files and folders

### ISSUE THREAD

**Issue by Romachamp** [zenangst/KeyboardCowboy#556]
Hello, I would like to see possibilities in the app for automating moving/renaming/creating/deleting files and folders for organizing them. Also, it would fit well with the existing open folder feature. Also, this kind of features can be extended as it is done in the Hazel app. 

Please, consider adding this feature.

Thank you, in advance!

**Comment by zenangst** [comment]
hey @Romachamp, what would be the trigger for this types of actions?
Or would you like them as commands? Unsure how I would go about implementing this without a lot of work and some (if not all) might be possible with some shell scripting today. Hence why I was curious about what would trigger the move/rename/creating and deleting of files.

**Comment by Romachamp** [comment]
I would want them just like commands. Triggers could be either rules like in hazel or opening folders, which you already have. Sure, it will require some work, but, in my opinion, it is core in the automation apps like this one.

**Comment by zenangst** [comment]
@Romachamp What do you mean by rules? What does that mean in practice?
I'm unfamiliar with Hazel

**Comment by zenangst** [comment]
@Romachamp Oh, I see... what you are asking for (when it comes to triggers) is file system event monitoring. So when you add a file to a watched folder, a set of commands should run?

**Comment by Romachamp** [comment]
Yes, exactly. But in your case, it could be even better, because you also have triggers for opening app windows/finder windows, which could be used as well. 

By rules, I meant something like If a file extension is .zip, then action(move/delete) or if file size is more than 500MB, then action(move/delete), etc. 

This is a feature, I really like in such apps.
